lib/ResourceFacade.js

```diff
@@ -1,5 +1,5 @@
 /**
- * A [Resource]{@ui5/project.Resource} with a different path than it's original
+ * A {@link @ui5/fs/Resource Resource} with a different path than it's original
```

Comment:
Do we have a preferred link style syntax over all UI5 Tooling projects?
In the most cases, I saw
`[Resource]{@link @ui5/fs/Resource}` but `{@link @ui5/fs/Resource Resource}` is also fine for me.

Reply:
This one seems to be transpiled properly to a link. The former was not. I'm not sure whether it's the jsdoc or the `@ui5` in the beginning, but after this change the link gets rendered properly

Reply:
The issue was the missing `@link`